js/src/ui/Form/AddressSelect/addressSelect.js

```diff
@@ -183,6 +182,12 @@ class AddressSelect extends Component {
     }
 
     const id = `addressSelect_${++currentId}`;
+    const ilHint = typeof hint === 'string' || !(hint && hint.props)
+      ? (hint || '')
+      : this.context.intl.formatMessage(
```

Comment:
The issue with this type of formatting is that the message will never be extracted. Hence we always opt for `FormattedMessage` directly (which does get extracted and then becomes available for translation). 

Not sure I understand what causes this issue and how to work around.